data/remnant missions.txt

```diff
@@ -655,149 +655,288 @@ mission "Remnant: Learn Sign Follow Up"
-			`	She taps her commlink, and moments later another Remnant appears from the scaffolding around a nearby Starling. The researcher straightens up and switches to a more stately melody. "Captain <last>, this is Prefect Taely. She is our most experienced engineer for dealing with alien tech."`
-			`	Taely steps forward and makes what seems to be a kind of welcoming gesture. "Greetings Captain. We appreciate your help in acquiring these samples. It looks like we are far behind the rest of humanity in some respects. We have a lot of work to do if we are to catch up with them."`
+			`As you land on a pad adjacent to the research center, a group of researchers mount some kind of weapon on a platform aimed down a firing range. The researchers retreat behind a thick barrier and press a button to activate the weapon. A flash of light erupts from the muzzle as the target downrange is cut in half. The barrel of the weapon raises into the air and the researchers rush out onto the range with tools and devices to inspect the damage. Noticing your arrival, one of the researchers veers off and heads towards you. "So, you are the one with the samples of humanity's laser weapons?" chants the researcher. "Always interesting to see new things." He turns and waves at a remnant standing near the edge of the landing pad with a camel hitched to a wagon.`
+			`	The researchers quickly unload the heavy lasers from your ship, and hand you a credstick with <payment>. "Thank you for bringing us something new to examine. We appreciate the breath of fresh air." With that, they quickly head off into a nearby lab, escorting their cargo of weaponry on the oddly archaic transportation.`
 
 
 
-mission "Remnant: Catalytic Ramscoop"
-	name "Retrieve Ramscoops for the Remnant"
-	description "A Remnant engineer on <planet> has offered to pay you <payment> in exchange for obtaining two catalytic ramscoops for them to study."
+mission "Remnant: Plasma Cannon"
+	name "Retrieve Plasma Cannons for the Remnant"
+	description "A Remnant engineer has offered to pay you <payment> in exchange for delivering two plasma cannons to a team on <planet>."
 	minor
-	source
+	source "Viminal"
+	destination
 		government "Remnant"
 	to offer
-		has "Remnant: Technology Available: offered"
-		not "Remnant: Plasma Cannons: active"
-		not "Remnant: Plasma Cannons: done"
-		not "Remnant: Heavy Laser: active"
-		not "Remnant: Heavy Laser: done"
-		random < 50
+		or
+			has "Remnant: Tech Retrieval: active"
+			has "Remnant: Tech Retrieval: done"
 	on offer
-		require "Catalytic Ramscoop"
+		require "Plasma Cannon"
 		conversation
-			`As you are walking through the spaceport, a woman comes up to you and introduces herself as an engineer at one of the Remnant development labs. She pauses awkwardly before beginning to chant: "As your ship is the only one we have seen in many decades that has outfits from ancestral space, we are intrigued by it. Could we take a closer look?"`
```

Comment:
Earlier the story says the player is the first through since the exodus, so shouldn't ""As your ship is the only one we have seen in many decades" read differently?  Maybe instead of "decades" say, "centuries?"

Reply:
This section is just being deleted.